dev/tasks/macros.jinja

```diff
@@ -362,8 +362,8 @@ on:
   {# use filter to cast to string and convert to lowercase to match yaml boolean #}
   {% set is_fork = (not is_upstream_b)|lower %}
 
-{% set r_release = {"ver": "4.2", "rt" : "42"} %}
-{% set r_oldrel = {"ver": "4.1", "rt" : "40"} %}
+{% set r_release = {"ver": "4.3", "rt" : "43"} %}
+{% set r_oldrel = {"ver": "4.2", "rt" : "42"} %}
```

Comment:
We can't update the 10.13 runner because there is no easy way to switch versions (versions of `rig` that contain 4.3 don't work on 10.13 anymore, even when build from source). We could of course special case that to run on 4.2/1 and update the other builds.